src/ceph.in

```diff
@@ -712,14 +712,14 @@ def ping_monitor(cluster_handle, name, timeout):
     if mon_id == '*':
         run_in_thread(cluster_handle.connect, timeout=timeout)
         for m in monids():
-            s = run_in_thread(cluster_handle.ping_monitor, m)
+            s = run_in_thread(cluster_handle.ping_monitor, m, timeout=timeout)
             if s is None:
                 print("mon.{0}".format(m) + '\n' + "Error connecting to monitor.")
             else:
-                print("mon.{0}".format(m) + '\n' + s)
+                print("mon.{0}".format(m) + '\n' + str(s))
```

Comment:
no need to `str(s)` explicitly.

Reply:
Not casting it would be erroneous. `+` only works for numeric types and strings -

```
 cat func.py 
def return_iterable():
    return -1, None, 'Interrupted!'

a = return_iterable()
print("type(a) - " + str(type(a)))
print("return value - " + a)
[rishabh@x220 py]$ python func.py 
type(a) - <type 'tuple'>
Traceback (most recent call last):
  File "func.py", line 6, in <module>
    print("return value - " + a)
TypeError: cannot concatenate 'str' and 'tuple' objects
```

Reply:
@rishabh-d-dave could you take a look at https://github.com/ceph/ceph/pull/24733/commits/3002ccba9a09540655e4d5e0d40bcbfa8024f24f . i think, if we don't return a tuple, then this problem does not exist anymore. am i right?

Reply:
Since we are getting rid of the tuple, we don't need this change.